src/cryptoadvance/specter/logic.py

```diff
@@ -583,23 +586,42 @@ def _check_change(self):
         self._dict["change_address"] = self.get_address(self._dict["change_index"], change=True)
         self._commit()
 
+    @property
+    def txlist(self):
```

Comment:
Can we have a documentation for this property? I guess my limited understanding is somehow:
''' The last 1000 transactions for that wallet but somehow filtered because ... ?! '''